### DISCUSSION TITLE
how to make this widget clickable ?

### DISCUSSION THREAD

**Discussion by jackywu** [web-infra-dev/rspress#949] [Q&A] [answered]
how to make this widget clickable and set a href for it ?

![Xnip2024-04-13_23-23-00](https://github.com/web-infra-dev/rspress/assets/587955/b2423339-9ff3-43bc-ab54-3d6f3529874c)

**Comment by xc2** [comment, marked as answer]
try `link`. https://rspress.dev/api/config/config-frontmatter.html#features

**Reply by jackywu** [reply]
thanks for replied. BTW, what does the term 'hero' means in this project?

**Reply by xc2** [reply]
"hero" is short for "hero section" or "landing hero". "hero" here means something "key" or "important". it's likely originates from [the film industry](https://foxxr.com/blog/epic-story-of-the-hero-image/) and has been widely adopted in the field of web design. It refers to the key and important section of a website that is immediately visible to users when they enter the site, to effectively showcase the website's content.

nowadays it's commonly referred to the design like https://dribbble.com/tags/landing-hero
